hooks/command

```diff
@@ -83,6 +83,11 @@ function run_docker_compose() {
     command+=(-f "docker-compose.yml")
   fi
 
+  # Append docker compose exconf file
```

Comment:
I'm confused, isn't this already supported by using multiple config files https://github.com/buildkite-plugins/docker-compose-buildkite-plugin#example ?